src/lib/plugins/live-cell/live-cell.tpl.html

```diff
@@ -0,0 +1,31 @@
+<ng-container [ngSwitch]="cell.column.type">
```

Comment:
this html should go under theme and for each column type we need to have own template like for body cell templates.